src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -892,11 +893,26 @@
 	end
 
 
+	function m.filterEmpty(dirs)
+		return table.translate(dirs, function(val)
+			if val and #val > 0 then
+				return val
+			else
+				return nil
+			end
+		end)
+	end
+
+
 	function m.additionalIncludeDirectories(cfg, includedirs)
 		if #includedirs > 0 then
 			local dirs = project.getrelative(cfg.project, includedirs)
-			dirs = path.translate(table.concat(dirs, ";"))
```

Comment:
Yes, I just ran into this locally as well ;) I accidentally mistook the "table.translate" in the m.filterEmpty method with a path.translate, so thought the m.filterEmpty already did the translation.... doh... I'll fix this one up, and move the filterEmpty method into table.lua